whatdid/ui_testhook/UiTestWindow.swift

```diff
@@ -119,6 +119,23 @@ fileprivate class AutocompleteComponent: TestComponent {
         adder(autocompleField)
         optionsStack.arrangedSubviews[1].leadingAnchor.constraint(equalTo: resultStack.arrangedSubviews[1].leadingAnchor).isActive = true
         
+        let separator = NSBox()
```

Comment:
This is to test #285. I committed it here accidentally, but I'm okay keeping it.